### DISCUSSION TITLE
Skywalking connection to Banyandb cluster failed

### DISCUSSION THREAD

**Discussion by kekekekekeshi** [apache/skywalking#12492] [Q&A]
**This is my docker compose configuration**
<img width="809" alt="image" src="https://github.com/user-attachments/assets/091c25b9-ccd8-4364-9abd-c2c4deb5d2ef">
**This is my health check result for the Banyandb cluster**
<img width="1747" alt="image" src="https://github.com/user-attachments/assets/1c712cd7-7ec3-4ae5-8e3d-470eb5d11627">
<img width="450" alt="image" src="https://github.com/user-attachments/assets/fb424a93-e04a-46c7-949b-dc340e947f83">
**This is my etcd user permission check**
<img width="1750" alt="image" src="https://github.com/user-attachments/assets/e32368e3-de64-4f9a-8341-1622dc376eb2">
**This is the result of my skywalking operation**
<img width="1746" alt="image" src="https://github.com/user-attachments/assets/c5596d0f-800f-48c5-bd76-42436af8d37a">

**Comment by hanahmily** [comment]
Hi, `/usr/bin/banyand-server data` is for the cluster mode. If you want to run banyandb as a single node, `/usr/bin/banyand-server standalone` works. Refer to [standalone installation](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/standalone.md). For setting up a cluster, use [cluster installation](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/cluster.md).

**Reply by kekekekekeshi** [reply]
> Hi, `/usr/bin/banyand-server data` is for the cluster mode. If you want to run banyandb as a single node, `/usr/bin/banyand-server standalone` works. Refer to [standalone installation](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/standalone.md). For setting up a cluster, use [cluster installation](https://github.com/apache/skywalking-banyandb/blob/main/docs/installation/cluster.md).

I have updated the detailed information, please help me see what's going on

**Reply by kekekekekeshi** [reply]
<img width="1099" alt="image" src="https://github.com/user-attachments/assets/6e9267f8-edcf-4e73-9d8c-08c4233e607b">
I seem to have already taken care of it. I've been tinkering with it for a long time, and I think this comment is poorly written. It should be clearly stated that the value of the SW-SORAGE_SANYANDB_TARGETS parameter can only be the liaison node.

**Reply by hanahmily** [reply]
Congratulations on solving the issue. The comment lacks clear instructions for connecting banyandb in cluster mode. I will submit a PR for this.

**Reply by hanahmily** [reply]
At the bottom of https://skywalking.apache.org/docs/main/next/en/setup/backend/storages/banyandb/, there are some references.
![image](https://github.com/user-attachments/assets/bcd24f7e-6c73-438d-9b74-ca814c9f8c65)

**Reply by kekekekekeshi** [reply]
> At the bottom of https://skywalking.apache.org/docs/main/next/en/setup/backend/storages/banyandb/, there are some references. ![image](https://private-user-images.githubusercontent.com/1960537/354228313-bcd24f7e-6c73-438d-9b74-ca814c9f8c65.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1NjA1NjYsIm5iZiI6MTcyMjU2MDI2NiwicGF0aCI6Ii8xOTYwNTM3LzM1NDIyODMxMy1iY2QyNGY3ZS02YzczLTQzOGQtOWI3NC1jYTgxNGM5ZjhjNjUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDJUMDA1NzQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzZlZTBjY2Q4MzE0ZmM3MzhmZmRkOGI0YjVkMjgyNDIwNTYwNWI1MjdiZTk1MzkzMDIwM2RmMjBjYTVmMGViOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.9f0jr6QQExSQLhIvcKQTo6YcpMitS335OZ0M-Wv2IpM)
**Okay, thank you, but the latest version is 0.6.1. When will 0.7 be released?**
<img width="1213" alt="image" src="https://github.com/user-attachments/assets/4617ff03-f853-4d61-9b74-727b58246b40">

**Reply by hanahmily** [reply]
Our current plan is at the end of this month.

**Reply by kekekekekeshi** [reply]
> Our current plan is at the end of this month.
ok
